tmp.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+$@
```

Comment:
Should this be deleted? why do we have this?

Reply:
oops this should definitely be deleted. That was my own testing that somehow got committed. Will fix.